client/components/Editor/schemas/table.ts

```diff
@@ -70,7 +70,7 @@ table.toDOM = (node: ProsemirrorNode) => {
 	return pruneFalsyValues([
 		spec[0],
 		{ id: node.attrs.id },
-		withValue(buildLabel(node), (label) => ['caption', { spellcheck: 'false' }, label]),
```

Comment:
We don't need `spellcheck` here because this function never returns a `DOMOutputSpec` that's consumed in contenteditable mode.